src/resources/PanelLayout.py

```diff
@@ -0,0 +1,24 @@
+from rich.layout import Layout
+
+
+class PanelLayout:
+    """Define panel layout"""
+
+    def __init__(self) -> None:
```

Comment:
Since you don't initialize this object anyways, I would suggest removing the init method